test/PngTests.re

```diff
@@ -74,4 +74,29 @@ describe("Png comparing", ({test, _}) => {
     expect.int(diffOfDiffPixels).toBe(0);
     expect.float(diffOfDiffPercentage).toBeCloseTo(0.0);
   });
+
+  test("does not count anti-aliased pixels as different", ({expect, _}) => {
+    let img1 = PureC_IO.IO.loadImage("test/test-images/antialiasing-on.png");
+    let img2 = PureC_IO.IO.loadImage("test/test-images/antialiasing-off.png");
+
+    let (_, diffPixels, diffPercentage) =
+      Diff.compare(img1, img2, ~outputDiffMask=true, ());
+
+    expect.int(diffPixels).toBe(38);
+    expect.float(diffPercentage).toBeCloseTo(0.095);
+  });
+
+  test("Diff of mask and no mask are equal", ({expect, _}) => {
+    let img1 = PureC_IO.IO.loadImage("test/test-images/antialiasing-on.png");
+    let img2 = PureC_IO.IO.loadImage("test/test-images/antialiasing-off.png");
+
+    let (_, diffPixels, diffPercentage) =
+      Diff.compare(img1, img2, ~outputDiffMask=false, ());
+
+    let (_, diffPixelsMask, diffPercentageMask) =
+      Diff.compare(img1, img2, ~outputDiffMask=true, ());
+
+    expect.int(diffPixels).toBe(diffPixelsMask);
+    expect.float(diffPercentage).toBeCloseTo(diffPercentageMask);
+  });
```

Comment:
This test fails, which it shouldn't, right? 
Do you have an idea why that would be?

Reply:
Well....the test was flawed. I painted the diff pixels on the image in the first diff run and diffed it again. 😄 
It is fixed now